ios/analytics.md

```diff
@@ -70,15 +69,15 @@ If you encounter an error message that begins `[!] Failed to connect to GitHub t
 
 2. Classes that call Amazon Pinpoint APIs must use the following import statements:
 
-    ```
+    ```swift
     /** start code copy **/
     import AWSCore
     import AWSPinpoint
     import AWSMobileClient
     /** end code copy **/
     ```
 
-3. Replace the return statement with following code into the `application(_:didFinishLaunchingWithOptions:)` method of your app's `AppDelegate.swift`.
+3. To receive send events with Amazon Pinpoint, you'll instantiate a Pinpoint instance. We recommend you do this during app startup, so you can use Pinpoint to record app launch analytics. Edit the `application(_:didFinishLaunchingWithOptions:)` method of your app's `AppDelegate.swift` by adding a `pinpoint` instance property, and initializing the Pinpoint client as shown below:
```

Comment:
Are send events for push notifications?

Reply:
D'oh! Should have been "to send events with Amazon Pinpoint"... Will fix in the next commit.